src/spacy/fortex/spacy/spacy_processors.py

```diff
@@ -28,7 +28,7 @@
 from forte.data.ontology import Annotation
 from forte.processors.base import PackProcessor, FixedSizeBatchProcessor
 from ft.onto.base_ontology import EntityMention, Sentence, Token, Dependency
-from ftx.medical.clinical import MedicalEntityMention, UMLSConceptLink
+from ftx.onto.clinical import MedicalEntityMention, UMLSConceptLink
```

Comment:
So for the demo pipeline code in `medical_pipeline.py`, we are able to use `ftx.onto.clinical` to import `MedicalEntityMention`, since `forte-medical` has to install `forte` first and it sets up `ftx.onto.clinical`. Similarly, if we use this wrapper in forte-medical, it should be able to do the same? Since `forte-medical`, by itself, does not redefine `MedicalEntityMention` in its ontology, just uses `forte`'s setup.

Reply:
I see, if that's the case it would be fine. I was thinking of the following scenario:

1. In Forte, we define `ftx.onto.clinical` as demo
2. In Forte medical, we redefine everything under `ftx.medical` so they are more consistent.
3. In the future, when Forte medical is mature, we may remove the demo and use Forte medical code.

The main reason is that having `ftx.onto` and `ftx.medical` might be a bit confusing to people.

Reply:
Forte-wrappers has a dependency on Forte and so does Forte-medical. Processors like Spacy, under `umls-linking` option need to have access to this ontology, and so does our medical pipeline. Hence, I think we will have to keep `MedicalEntityMention` and `UMLSConceptLink` ontologies in Forte and won't be able to remove it from there, given its a dependency to `SpacyProcessor`(in Forte-wrappers). We should be good if we don't redefine anything in Forte-medical (which we aren't at the present).

Reply:
Yeah, but this will leave two namespaces. 

My suggestion is that both forte-wrapper and forte-medical don't have to have a hard dependency on the ontology, we can provide the ontology name in the config as I suggested above.

Reply:
yeah, the trick is just like you said, since you have the class name, you can create an instance (we do have [utilities for it](https://github.com/asyml/forte/blob/master/forte/utils/utils.py#L132)), and then you can use it normally.

So basically we decide which ontology entry we use at run time, which means we can use the ontology we want and `SpacyProcessor` don't have to know about it. This make the forte wrapper library more flexible to other libraries, which is a thing we plan to do more later (for other processors).

Reply:
I think there is an working example [here](https://github.com/asyml/forte-wrappers/blob/main/src/huggingface/fortex/huggingface/zero_shot_classifier.py#L76)